test/reaping-faulty-nodes.js

```diff
@@ -278,7 +278,7 @@ function pingNotTombstoneState(t, tc) {
         return ping.type === events.Types.Ping
             && ping.direction === 'request'
             && ping.receiver !== tc.fakeNodes[0].getHostPort() // ignore nodeIx=0
-            && ping.body.changes[0].status !== 'alive'  // ignore old ping
+            && ! (ping.body.changes[0] && ping.body.changes[0].status === 'alive')
```

Comment:
Two reasons:

1. The line is explained above already in a separate paragraph -- the comment here predates that explanation.
2. The line length becomes a bit too long for my taste.